### PR TITLE
Add DDI 3.3 schemas to the server

### PR DESCRIPTION
**Pull request by matthew-morris-cessda**
This allows DDI 3.3 documents to be validated against the XML schema.

### CODE REVIEW

File: src/main/resources/static/schemas/lifecycle/3.2/XHTML/xhtml-charent-1.xsd [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+           xmlns="http://www.w3.org/1999/xhtml">
+
+    <xs:annotation>
+        <xs:documentation>
+            Character Entities for XHTML
+            This is the XML Schema Character Entities module for XHTML
+
+            This module declares the set of character entities for XHTML,
+            including the Latin 1, Symbol and Special character collections.
+            XML Schema does not support Entities, hence Entities are enable
```
Comment (john-shepherdson) [inline src/main/resources/static/schemas/lifecycle/3.2/XHTML/xhtml-charent-1.xsd:33]: enable -> enabled

File: src/main/resources/static/schemas/lifecycle/3.2/XHTML/xhtml-inlstyle-1.xsd [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="http://www.w3.org/1999/xhtml"
+           xmlns="http://www.w3.org/1999/xhtml">
+
+    <xs:annotation>
+        <xs:documentation>
+            Inline Style module
+            This is the XML Schema Inline Style module for XHTML
+
+            * styloe attribute
```
Comment (john-shepherdson) [inline src/main/resources/static/schemas/lifecycle/3.2/XHTML/xhtml-inlstyle-1.xsd:11]: styloe -> style

File: src/main/resources/static/schemas/lifecycle/3.2/XHTML/xhtml-notations-1.xsd [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+            $Id$
+        </xs:documentation>
+        <xs:documentation source="xhtml-copyright-1.xsd"/>
+    </xs:annotation>
+
+    <xs:annotation>
+        <xs:documentation>
+            Notations module
+            Defines the XHTML notations, many of these imported from
+            other specifications and standards. When an existing FPI is
```
Comment (john-shepherdson) [inline src/main/resources/static/schemas/lifecycle/3.2/XHTML/xhtml-notations-1.xsd:19]: What is FPI?

File: src/main/resources/static/schemas/lifecycle/3.3/XHTML/xhtml-charent-1.xsd [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+           xmlns="http://www.w3.org/1999/xhtml">
+
+  <xs:annotation>
+    <xs:documentation>
+      Character Entities for XHTML    
+      This is the XML Schema Character Entities module for XHTML
+
+      This module declares the set of character entities for XHTML,
+      including the Latin 1, Symbol and Special character collections.
+      XML Schema does not support Entities, hence Entities are enable
```
Comment (john-shepherdson) [inline src/main/resources/static/schemas/lifecycle/3.3/XHTML/xhtml-charent-1.xsd:33]: enable -> enabled

File: src/main/resources/static/schemas/lifecycle/3.3/XHTML/xhtml-framework-1.xsd [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+      </xs:documentation>
+      <xs:documentation
+          source="http://www.w3.org/TR/2001/REC-xhtml-modularization-20010410/abstraction.html#s_common_attrtypes"/>
+    </xs:annotation>
+  </xs:include>
+
+  <xs:include schemaLocation="xhtml-attribs-1.xsd">
+    <xs:annotation>
+      <xs:documentation>
+        This module defines Common attributes for XHTML
```
Comment (john-shepherdson) [inline src/main/resources/static/schemas/lifecycle/3.3/XHTML/xhtml-framework-1.xsd:52]: attributes -> Attributes (for consistency)

File: src/main/resources/static/schemas/lifecycle/3.3/XHTML/xhtml-notations-1.xsd [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+      $Id$
+    </xs:documentation>
+    <xs:documentation source="xhtml-copyright-1.xsd"/>
+  </xs:annotation>
+  
+  <xs:annotation>
+    <xs:documentation>
+      Notations module
+      Defines the XHTML notations, many of these imported from 
+      other specifications and standards. When an existing FPI is
```
Comment (john-shepherdson) [inline src/main/resources/static/schemas/lifecycle/3.3/XHTML/xhtml-notations-1.xsd:19]: FPI in full

**Review by john-shepherdson — COMMENTED** [review]
Approved, subject to responses to comments regarding typos

**Comment by matthew-morris-cessda** [comment]
The schema is directly copies from the official DDI release, so regardless of the mistakes I don't think we should modify it

**Comment by john-shepherdson** [comment]
OK, it was not clear to me that all the files are copied from 3rd party sources

**Review by john-shepherdson — APPROVED** [review]

**Comment by matthew-morris-cessda** [comment]
This applies to all of the XML schemas. I'll add a note specifying the source of these schemas. Maybe DDI would like to know of the mistakes.

**Pull request merged**